### PR TITLE
Add Squad.app v2.7.0

### PR DESCRIPTION
**Pull request by andrewschreiber**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/squad.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'squad' do
+  version '2.7.0'
+  sha256 'abbc773046f0d303f3381e131a16577911028620ef6741b822f56fddbfd3a594'
+
+  url "https://andrewschreiber.github.io/releases/squad_#{version.gsub('.', '')}.zip"
+  appcast 'https://andrewschreiber.github.io/squadappcast.xml',
+          checkpoint: '0242efe468fb4018e319b97796263a21951711d198e4069887d939f955968faa'
+  name 'Squad'
+  homepage 'https://andrewschreiber.github.io'
```
Comment (miccal) [inline Casks/squad.rb]: There is no information about this `app` on this homepage?

Comment (andrewschreiber) [inline Casks/squad.rb]: Adding a blog post with photos now.

Comment (andrewschreiber) [inline Casks/squad.rb]: Blog post added!

Comment (miccal) [inline Casks/squad.rb]: Thank you.

**Comment by miccal** [comment]
Ping @vitorgalvao for approval.

File: Casks/squad.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'squad' do
+  version '2.7.0'
+  sha256 'abbc773046f0d303f3381e131a16577911028620ef6741b822f56fddbfd3a594'
+
+  url "https://andrewschreiber.github.io/releases/squad_#{version.gsub('.', '')}.zip"
```
Comment (Numbermaniac) [inline Casks/squad.rb]: Would `#{version.no_dots}` work here?

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**